### PR TITLE
lib/downloader: Fix progress bar length on older CrOS terminals

### PR DESCRIPTION
**Pull request by supechicken**
On older CrOS terminals (mainly `i686`), the progress bar will overflow since a width bug will occur when using the box-drawing char with bold character color form (`1;36m`)

Changed the front/background color code to its non-bold equivalent in this PR

### Run the following to get this pull request's changes locally for testing.
```shell
CREW_TESTING=1 CREW_TESTING_REPO=https://github.com/supechicken/chromebrew CREW_TESTING_BRANCH=fix_progbar_color crew update
```

### CODE REVIEW

File: lib/progress_bar.rb [hunk 1]
```diff
@@ -11,10 +11,9 @@ def initialize(total_size)
     # character used to fill the progress bar, one of the box-drawing character in unicode
     @bar_char = "\u2501"
 
-    # color scheme of progress bar, can be changed
-    # see color.rb for more available colors
-    @bar_front_color = :lightcyan
-    @bar_bg_color = :gray
+    # color scheme of progress bar (in ANSI color codes), can be changed
+    @bar_front_color = [96, 0]
+    @bar_bg_color = [90, 0]
```
Comment (uberhacker) [inline lib/progress_bar.rb:16]: Why the change?  What is wrong with the more descriptive symbol notation?

Comment (supechicken) [reply lib/progress_bar.rb:16]: Since the `90-99` color code does not exist in `color.rb`

Comment (supechicken) [reply lib/progress_bar.rb:16]: (thinking what should they name if we decided to add them to `color.rb` since `light*` prefix are already assigned to the bold form

Comment (uberhacker) [reply lib/progress_bar.rb:16]: Prefix with `lite`?  Add to `color.rb`?

**Review by uberhacker — APPROVED** [review]
Testing works fine.
